python/looker/rtl/transport.py

```diff
@@ -30,9 +30,20 @@ class TransportSettings:
     base_url: str
     api_version: str
     headers: Optional[MutableMapping[str, str]] = None
+    """API-versioned base endpoint"""
```

Comment:
the [styleguide](http://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments) doesn't address this specifically but I think these should be regular comments rather than docstring-like `str` objects in the code.